spirv_reflect.c

```diff
@@ -1227,26 +1254,74 @@ static SpvReflectResult ParseFunction(
         CHECKED_READU32(p_parser, p_node->word_offset + 3,
                         p_func->callees[p_func->callee_count]);
         (++p_func->callee_count);
-      }
-      break;
-      case SpvOpLoad:
+      } break;
       case SpvOpAccessChain:
       case SpvOpInBoundsAccessChain:
       case SpvOpPtrAccessChain:
       case SpvOpArrayLength:
       case SpvOpGenericPtrMemSemantics:
-      case SpvOpInBoundsPtrAccessChain:
+      case SpvOpInBoundsPtrAccessChain: {
+        CHECKED_READU32(p_parser, p_node->word_offset + 3,
+                        p_func->accessed_ptrs[p_func->accessed_ptr_count]);
+
+        CHECKED_READU32(
+            p_parser, p_node->word_offset + 3,
+            p_func->accessed_variable_ptrs[p_func->accessed_ptr_count]
+                .variable_ptr);
```

Comment:
For readability (as readable as C89 can be), can the member access just stay on the same line as their parent?